subworkflows/local/pggb.nf

```diff
@@ -0,0 +1,140 @@
+//
+// Run the default PanGenome Graph Builder (PGGB) pipeline
+//
+
+include { WFMASH as WFMASH_MAP_ALIGN      } from '../../modules/nf-core/wfmash/main'
+include { WFMASH as WFMASH_MAP            } from '../../modules/nf-core/wfmash/main'
+include { WFMASH as WFMASH_ALIGN          } from '../../modules/nf-core/wfmash/main'
+include { SEQWISH_INDUCE as SEQWISH       } from '../../modules/nf-core/seqwish/induce/main'
+include { SMOOTHXG                        } from '../../modules/nf-core/smoothxg/main'
+include { GFAFFIX                         } from '../../modules/nf-core/gfaffix/main'
+include { ODGI_BUILD                      } from '../../modules/nf-core/odgi/build/main'
+include { ODGI_UNCHOP                     } from '../../modules/nf-core/odgi/unchop/main'
+include { ODGI_SORT                       } from '../../modules/nf-core/odgi/sort/main'
+include { ODGI_VIEW                       } from '../../modules/nf-core/odgi/view/main'
+
```

Comment:
```
```

Reply:
Sorry I just see an empty comment here, what did you have in mind?